tools/fuzzer/main.c

```diff
@@ -76,7 +76,8 @@ static struct fuzz_platform *platform[] = {
 		&cht_platform,
 		&bsw_platform,
 		&hsw_platform,
-		&bdw_platform
+		&bdw_platform,
+		&imx8_platform
```

Comment:
Can't we have a comma at the end of this line so that future changes won't have to change two lines?

Reply:
@dianacretu yes, we need to have comma after the last entry in order to have a smaller diff next time we modify this array.
So, you should add:
```
		&imx8_platform,
```

Reply:
@dianacretu this comment isn't yet fixed! Will talk about it more on our next meeting.